art/classifiers/scikitlearn.py

```diff
@@ -860,11 +860,14 @@ def __init__(self, model, clip_values=None, defences=None, preprocessing=(0, 1))
         from sklearn.svm import SVC, LinearSVC
 
         if not isinstance(model, SVC) and not isinstance(model, LinearSVC):
-            raise TypeError('Model must be of type sklearn.svm.SVC or sklearn.svm.LinearSVC')
+            raise TypeError('Model must be of type sklearn.svm.SVC or sklearn.svm.LinearSVC. Found type {}'
+                            .format(type(model)))
 
         super(ScikitlearnSVC, self).__init__(model=model, clip_values=clip_values, defences=defences,
                                              preprocessing=preprocessing)
         self._model = model
+        self.kernel_func = self._kernel_func()
+        self.kernel_grad = self._kernel_gradient_func
```

Comment:
What is the reason for the changes in this and the previous line? Would it be possible to just use the function names on the right hand side?

Reply:
For `kernel_grad` yes it's essentially just a renaming, but `_kernel_func()` is a function that returns a function (the appropriate kernel). I'll still add the underscores.